.buildkite/scripts/integration_tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+export WORKSPACE=`pwd`
+
+vault kv get -format=json -field=data kv/ci-shared/observability-ingest/cloud/gcp > ./gcp.json
```

Comment:
Usually we're using the pre-command hook (script) to get the secrets or credentials and also ensure that they redacted.

Could this be moved to that hook, and use some of the suffixes to ensure that is redacted in the output ? 

https://buildkite.com/docs/pipelines/managing-log-output#redacted-environment-variables